### DISCUSSION TITLE
Handling `X-Forwarded-For` from CloudFlare (tunnel)?

### DISCUSSION THREAD

**Discussion by MikeChongCan** [dani-garcia/vaultwarden#2446] [General]
Hello!

I'm using CloudFlare Tunnel for VaultWarden.
However, the IP address shown in the email about signing in from a new device is always internal IP instead of `X-Forwarded-For` http header.

Is there any way to configure this one?

**Comment by MikeChongCan** [comment]
Maybe `IP_HEADER=` <https://github.dev/dani-garcia/vaultwarden/blob/3abf173d8954c982383285b29b39d15043e974cb/.env.template#L46> can help?

**Reply by BlackDex** [reply]
That is the correct one.
